open_spiel/games/maedn_test.cc

```diff
@@ -0,0 +1,376 @@
+// Copyright 2019 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "open_spiel/games/maedn.h"
+
+#include "open_spiel/spiel.h"
+#include "open_spiel/spiel_utils.h"
+#include "open_spiel/tests/basic_tests.h"
+
+namespace open_spiel {
+namespace maedn {
+namespace {
+
+namespace testing = open_spiel::testing;
+
+void BasicMaednTests() {
+  testing::LoadGameTest("maedn");
+
+  std::shared_ptr<const Game> game = LoadGame("maedn", 
+    {{"players", GameParameter(2)},
+    {"twoPlayersOpposite", GameParameter(true)}});  
+
+  testing::RandomSimTest(*game, 100);
+
+  game = LoadGame("maedn", 
```

Comment:
Pretty sure there's no need to create another instance of the game with the same parameters.